packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/index.tsx

```diff
@@ -252,48 +252,63 @@ const DialogHeader = ({ comment, hasShadow }) => {
   const closeDialog = () => actions.comments.closeComment();
 
   return (
-    <Stack
-      align="center"
-      justify="space-between"
-      padding={4}
-      paddingRight={2}
-      marginBottom={2}
-      css={css({
-        boxShadow: theme =>
-          hasShadow
-            ? `0px 32px 32px ${theme.colors.dialog.background}`
-            : 'none',
-      })}
-    >
-      <Text size={3} weight="bold">
-        Comment
-      </Text>
-      <Stack align="center">
-        <IconButton
-          onClick={() =>
-            actions.comments.resolveComment({
-              commentId: comment.id,
-              isResolved: !comment.isResolved,
-            })
-          }
-          name="check"
-          size={14}
-          title="Resolve Comment"
-          css={css({
-            transition: 'color',
-            transitionDuration: theme => theme.speeds[1],
-            color: comment.isResolved ? 'green' : 'mutedForeground',
-            ':hover:not(:disabled), :focus:not(:disabled)': {
-              color: comment.isResolved ? 'green' : 'foreground',
-            },
-          })}
-        />
-        <IconButton
-          name="cross"
-          size={10}
-          title="Close comment dialog"
-          onClick={closeDialog}
-        />
+    <Stack direction="vertical" css={{ position: 'relative' }}>
+      <div
+        css={css({
+          position: 'absolute',
+          height: '6px',
+          width: '100%',
+          backgroundColor: 'dialog.background',
+          bottom: '6px',
+          zIndex: 0,
```

Comment:
if this is z-index 0, it wouldn't hang over the content to create the fade-into-background effect that we want. it would be under the content